Testing/VelaptorTests/Content/Caching/TextureCacheTests.cs

```diff
@@ -530,34 +540,36 @@ public void ShutDownNotification_WhenReceived_DisposesOfTextures()
             MockTextureCreation(mockTextureA.Object, "textureA", texturePathA);
             MockTextureCreation(mockTextureB.Object, "textureB", texturePathB);
 
-            this.mockShutDownReactable.Setup(m => m.Subscribe(It.IsAny<IReactor<ShutDownData>>()))
-                .Returns(this.mockShutDownUnsubscriber.Object)
-                .Callback<IReactor<ShutDownData>>(reactor =>
-                {
-                    if (reactor is null)
-                    {
-                        Assert.True(false, "Shutdown reactable subscription failed.  Reactor is null.");
-                    }
-
-                    shutDownReactor = reactor;
-                });
-
             var cache = CreateCache();
             cache.GetItem(texturePathA);
             cache.GetItem(texturePathB);
 
             // Act
-            shutDownReactor?.OnNext(default);
-            shutDownReactor?.OnNext(default);
+            this.shutDownReactor?.OnNext(default);
+            this.shutDownReactor?.OnNext(default);
 
             // Assert
             this.mockDisposeTextureReactable
                 .Verify(m =>
-                    m.PushNotification(new DisposeTextureData(11u), false), Times.Once);
+                    m.PushNotification(new DisposeTextureData(11u)), Times.Once);
             this.mockDisposeTextureReactable
                 .Verify(m =>
-                    m.PushNotification(new DisposeTextureData(22u), false), Times.Once);
-            this.mockShutDownUnsubscriber.Verify(m => m.Dispose(), Times.Once);
+                    m.PushNotification(new DisposeTextureData(22u)), Times.Once);
+        }
+        #endregion
+
+        #region Indirect Internal Tests
+        [Fact]
+        public void ShutDownReactable_WhenNotificationsEnded_DisposesOfUnsubscriber()
```

Comment:
Refactor unit test name to below:

```suggestion
        public void ShutDownReactable_WhenNotificationsEnd_DisposesOfUnsubscriber()
```